Mathlib/Order/WellFounded.lean

```diff
@@ -286,3 +286,19 @@ theorem WellFounded.induction_bot {α} {r : α → α → Prop} (hwf : WellFound
 #align well_founded.induction_bot WellFounded.induction_bot
 
 end Induction
+
+section WellFoundedLT
+
+theorem WellFoundedLT.of_strictMono [Preorder α] [Preorder β] [WellFoundedLT β] {f : α → β}
+    (hf : StrictMono f) : WellFoundedLT α where
```

Comment:
Can you provide the `WellFoundedGT` and `StrictAnti` versions? Should be three more lemmas in total